src/site/content/en/learn/forms/forms.11tydata.js

```diff
@@ -0,0 +1,38 @@
+module.exports = function () {
+  // The key is used to look up the data for the course in _data.
+  // e.g. A course with a key of 'a11y' would have a corresponding
+  // _data/courses/a11y directory.
+  const projectKey = 'forms';
+
+  return {
+    layout: 'course',
+    projectKey,
+    searchTag: `course-${projectKey}`,
+    // Exclude course content from the /tags/ pages.
+    excludeFromTags: true,
+    // Exclude course content from the /authors/ pages.
+    excludeFromAuthors: true,
+    eleventyComputed: {
+      tags: (data) => {
+        const {searchTag} = data;
+        let tags = data?.courses?.[projectKey]?.meta?.tags || [];
+        tags = [...tags, ...data.tags];
+        if (!tags.includes(searchTag)) {
+          tags.push(searchTag);
+        }
+        return tags;
+      },
+      thumbnail: (data) => {
+        // Use thumbnail defined in the frontmatter or in the meta.yml file.
+        return data.thumbnail || data?.courses?.[projectKey]?.meta?.thumbnail;
+      },
+      // Use this flag to prevent the course from showing up in production.
+      permalink: (data) => {
+        if (process.env.ELEVENTY_ENV === 'prod') {
+          return false;
+        }
+        return (data.page.filePathStem + '.html').replace(/^\/en\//, '/');
```

Comment:
So this is basically a TODO right - we're not publishing this yet?

I'm a bit confused as to why the second non-prod case is needed. Is this just the "default" behavior that we don't get because there's an override here?

Reply:
> So this is basically a TODO right - we're not publishing this yet?

Yes. This lets folks check in content without it actually going live.

> I'm a bit confused as to why the second non-prod case is needed. Is this just the "default" behavior that we don't get because there's an override here?

Yeah, if I recall correctly, I had to add this otherwise the page's permalink did not work correctly (I think it ended up including /en/). I had a comment for this the first time I used this trick: https://github.com/GoogleChrome/web.dev/pull/5304/files#diff-383d1becf08325fc7dd2a0672ea3de9dd434dabdff8325c74131f392975c1f20L23

In theory our output-permalink.js thing that Michael wrote _should_ be stripping the /en/. But something about the order that these things run in made that not happen. I'll include a comment so folks understand what this is doing.

Reply:
Yeah, this feels gross. But it's fine for now.